### PR TITLE
Update extension-to-physical-entity-mib.md

### PR DESCRIPTION
**Pull request by keboliu**

### CODE REVIEW

File: doc/snmp/extension-to-physical-entity-mib.md [hunk 1]
```diff
@@ -372,3 +375,68 @@ New test cases will be added to cover the new MIB entries:
...
 4. Remove/Add DB entries from related tables to see whether MIB info can be correctly updated.
 5. Currently, each platform API is tested by sonic-mgmt, see [here](https://github.com/Azure/sonic-mgmt/tree/master/tests/platform_tests/api). We will add regression test case for each newly added platform API to verify them.
+
+
+## 2. Entity Sensor MIB Extension
+
+### 2.1 Entity Sensor MIB Extension Scope
+
+Currently SONiC already have a RFC3433 Entity sensor MIB implementation,  Transceiver embeded sensors like temperature, voltage, rx_power, tx_power and tx_bias are alreay included. After Entity MIB was extended as desribed in Section 1, Entity MIB will also be extened to cover more sensors. 
+Thermal sensors besides the Transceiver embeded thermal sensors which avalaible in the STATE_DB will all be add to the Entity Sensor MIB support.
```
Comment (Junchao-Mellanox) [inline doc/snmp/extension-to-physical-entity-mib.md:385]: all be add -> all be added

File: doc/snmp/extension-to-physical-entity-mib.md [hunk 2]
```diff
@@ -372,3 +375,68 @@ New test cases will be added to cover the new MIB entries:
...
+
+    TYPE = EntitySensorDataType.UNKNOWN
+    SCALE = EntitySensorDataScale.UNITS
+    PRECISION = 0
+
+Thermal sensors:
+
+    TYPE = EntitySensorDataType.CELSIUS
+    SCALE = EntitySensorDataScale.UNITS
+    PRECISION = 1
```
Comment (Junchao-Mellanox) [inline doc/snmp/extension-to-physical-entity-mib.md:405]: The temperature value in sysfs is usually something like 40123, and the value in DB will be 40.123. So maybe the PRECISION could be extended to 3.

File: doc/snmp/extension-to-physical-entity-mib.md [hunk 3]
```diff
@@ -372,3 +375,68 @@ New test cases will be added to cover the new MIB entries:
...
+
+PSU Temp Sensor:
+    
+    TYPE = EntitySensorDataType.CELSIUS
+    SCALE = EntitySensorDataScale.UNITS
+    PRECISION = 1
```
Comment (Junchao-Mellanox) [inline doc/snmp/extension-to-physical-entity-mib.md:411]: The temperature value in sysfs is usually something like 40123, and the value in DB will be 40.123. So maybe the PRECISION could be extended to 3.

Comment (keboliu) [reply doc/snmp/extension-to-physical-entity-mib.md:411]: do you think that we need to have a separated PSU temp sensor? it should be included in the thermal sensors? or not?

Comment (Junchao-Mellanox) [reply doc/snmp/extension-to-physical-entity-mib.md]: Yes, i think PSU temp sensor should be treated separately. The reason is that: in physical entity mib, PSU temp sensor is a child of PSU, and the physical index of PSU temp sensor is calculated based on PSU.

**Pull request closed without merging**